Mathlib/Data/Int/Cast/Defs.lean

```diff
@@ -42,6 +42,14 @@ class AddGroupWithOne (R : Type u) extends IntCast R, AddMonoidWithOne R, AddGro
   of the canonical homomorphism `ℕ → R`. -/
   intCast_negSucc : ∀ n : ℕ, intCast (Int.negSucc n) = - Nat.cast (n + 1) := by intros; rfl
 
+attribute [instance 50] AddGroupWithOne.toAddGroup
+attribute [instance 0] AddGroupWithOne.toNeg
+attribute [instance 0] AddGroupWithOne.toSub
+
 /-- An `AddCommGroupWithOne` is an `AddGroupWithOne` satisfying `a + b = b + a`. -/
 class AddCommGroupWithOne (R : Type u)
   extends AddCommGroup R, AddGroupWithOne R, AddCommMonoidWithOne R
+
+attribute [instance 50] AddCommGroupWithOne.toAddCommGroup
+attribute [instance 0] AddCommGroupWithOne.toNatCast
+attribute [instance 0] AddCommGroupWithOne.toOne
```

Comment:
Are these still needed? I think they're not instances at all any more in new Lean.